### ISSUE TITLE
Still no updates?

### ISSUE THREAD

**Issue by CyanDragonS** [kiwibrowser/src.next#1185]
almost 4 months no updates....

**Comment by PawelGu** [comment]
I also hope there will be updates. It's the - only - browser based on Chrome with extension support on Android. I like the clean standard Chrome experience degoogled and with all ad-blockingand possibilities. Sync via Floccus is also possible (-: And with Mozilla going the crap route...

**Comment by CyanDragonS** [comment]
Some web pages that use new chrome features are no longer visible :(, it's too bad.
I'd have to change my browser to edge canary and install extensions from dev setting.

**Comment by CyanDragonS** [comment]
Some web pages that use new chrome features are no longer visible :(, it's too bad.
I'd have to change my browser to edge canary and install extensions from dev setting.

@PawelGu Now edge canary also can install extensions from dev settings. You can download it from google play.

**Comment by lje** [comment]
Thanks for pointing out Edge Canary, so there is another Chromium based browser supporting extensions.
Kiwi seems to be abandoned, continuing to use it is a major security risk.

**Comment by PawelGu** [comment]
> Some web pages that use new chrome features are no longer visible :(, it's too bad. I'd have to change my browser to edge canary and install extensions from dev setting.
> 
> @PawelGu Now edge canary also can install extensions from dev settings. You can download it from google play.
> 
> Thanks for all developers made such a great app, but it's little bit slow to update, so i have to change a browser app temporarily and wait for update.

Great, I didn't have this one on my radar. I tried it out and it seems to work well except my mobile gets unusually hot with some of the extensions installed by extension id through developer options. Well it's very new and alpha so I hope it eventually will work out. Only problem then still will be Microsofts heavy telemetry.

**Comment by CyanDragonS** [comment]
@PawelGu 
Yes, this is a alpha version, some plugins still have problems, and even some times the screen will go blank, but it still works fine, my phone has no heating problems, presumably it may be that the plugin is not compatible with browsers?

Plugin compatibility work is complicated, so I have a feeling that it will be in canary version for a long time, and will not be open for download in the stable version

**Comment by lje** [comment]
> Only problem then still will be Microsofts heavy telemetry.

There are 3 options now:
1. use Fennec from F-Droid, which is Firefox with telementry removed and about:config enabled, supports Firefox extensions but may have less website compatibility than Chromium and no tab-based process isolation
2. use Edge Canary with MS telemetry
3. use abandoned Kiwi with known security holes

Chose the lesser evil.

**Comment by PawelGu** [comment]
@CyanDragonS 
Probably, not to mention extensions only arrived half a year ago for Edge Canary, maybe we will have the honor of a beta implementation next year.

@lje 
1. -> Better 'Mull'.
